src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -860,7 +860,7 @@ async function openNewNotebook(
 export class NewNotebookTask extends Task {
 	public static ID = 'newNotebook';
 	public static LABEL = localize('newNotebookTask.newNotebook', "New Notebook");
-	public static ICON = 'new-notebook';
+	public static ICON = 'dashboard-new-notebook';
```

Comment:
Is there a reason you're changing this name? Ideally we shouldn't have icons scoped to specific places like this - so I'd even go so far as to just have `notebook` as the icon name instead of `new-notebook` (since there's nothing indicating it's "new" in the icon)

Reply:
NewNotebookAction already claimed new-notebook, so I didn't want there to be any css collisions. I'll simplify it to just "notebook" and throw it in with the common icons